gradle.properties

```diff
@@ -138,7 +138,7 @@ ANDROID_COMPILE_SDK_VERSION=34
 
 ANDROID_NDK_VERSION=23.0.7599858
 
-ANDROID_GRADLE_PLUGIN_VERSION=8.3.0
+ANDROID_GRADLE_PLUGIN_VERSION=8.5.0
```

Comment:
minor change - bumped AGP. There were security and performance fixes. On my device the performance  improved dramatically